.eslintrc

```diff
@@ -91,8 +91,6 @@
     "comma-style": [2, "last"],
     // Warn about cyclomatic complexity in functions.
     "complexity": 2,
-    // Require return statements to either always or never specify values.
-    "consistent-return": 2,
```

Comment:
oops, meant to open a separate PR. completely agree that this makes us inconsistent w/ devtools. I feel like we can diverge in this way as we have `flow` for types, but I don't feel strongly.